run-unit-test.sh

```diff
@@ -15,5 +15,6 @@ virtualenv env
 . env/bin/activate
 pip install -e .
 pip install nose mock coverage
+rm .coverage
```

Comment:
That's not the point, though.  The point is that .coverage is cumulative, and so - unless it's deleted before each test run - it's easy to make a change that _loses_ some existing code coverage, and not to notice that that coverage has been lost.